### PR TITLE
Typo in doc (IO () should be return ())

### PR DESCRIPTION
**Pull request by nomeata**

### CODE REVIEW

File: src/Test/HUnit/Lang.hs [hunk 1]
```diff
@@ -49,7 +49,7 @@ location = case reverse callStack of
 --
 -- @
 --    if conditionIsMet
---        then IO ()
+--        then return ()
 --        else assertFailure msg
```
Comment (sol) [inline src/Test/HUnit/Lang.hs:53]: @nomeata Hey! Thanks for opening this PR.  If we are already at it, how about making it more idiomatic, e.g. `unless  conditionIsMet $ assertFailure message`?

Comment (nomeata) [reply src/Test/HUnit/Lang.hs:53]: 🤷. Some readers might not actually know about `Control.Monad.unless`.

Comment (sol) [reply src/Test/HUnit/Lang.hs:53]: I would argue that then it's the more important that they learn about it :smile:

Comment (sol) [reply src/Test/HUnit/Lang.hs:53]: On second though, the statement in the docs is no longer true, e.g. `assertEqual` can't be expressed by using `assertFailure`.  I'm going to delete that whole sentence.

Comment (sol) [reply src/Test/HUnit/Lang.hs:53]: piggybacked into #28

**Pull request closed without merging**